draft-ietf-quic-transport.md

```diff
@@ -6824,27 +6807,27 @@ accurate representation of any deployed usage.
 ### Permanent Registrations {#iana-permanent}
 
 Permanent registrations in QUIC registries use the Specification Required policy
-{{!RFC8126}}, unless otherwise specified.  The designated expert(s) verify that
-a specification exists and is readily accessible.  Expert(s) are encouraged to
-be biased towards approving registrations unless they are abusive, frivolous, or
-actively harmful (not merely aesthetically displeasing, or architecturally
+({{!RFC8126}}), unless otherwise specified.  The designated expert(s) verify
+that a specification exists and is readily accessible.  Expert(s) are encouraged
+to be biased towards approving registrations unless they are abusive, frivolous,
+or actively harmful (not merely aesthetically displeasing, or architecturally
 dubious).  The creation of a registry MAY specify additional constraints on
 permanent registrations.
 
-The creation of a registries MAY identify a range of codepoints where
+The creation of a registry MAY identify a range of codepoints where
 registrations are governed by a different registration policy.  For instance,
 the registries for 62-bit codepoints in this document have stricter policies for
 codepoints in the range from 0 to 63.
 
 Any stricter requirements for permanent registrations do not prevent provisional
 registrations for affected codepoints.  For instance, a provisional registration
-for a frame type {{iana-frames}} of 61 could be requested.
+for a frame type ({{iana-frames}}) of 61 could be requested.
 
 All registrations made by Standards Track publications MUST be permanent.
 
 All registrations in this document are assigned a permanent status and list as
 contact both the IESG (ietf@ietf.org) and the QUIC working group
-([quic@ietf.org](mailto:quic@ietf.org)).
```

Comment:
Hyperlinking results in the e-mail address being included twice in the text output.